service/client_state.hh

```diff
@@ -88,10 +89,17 @@ public:
     };
 private:
     client_state(const client_state* cs, seastar::sharded<auth::service>* auth_service)
-            : _keyspace(cs->_keyspace),  _user(cs->_user), _auth_state(cs->_auth_state),
-              _is_internal(cs->_is_internal), _is_thrift(cs->_is_thrift), _remote_address(cs->_remote_address),
-              _auth_service(auth_service ? &auth_service->local() : nullptr),
-              _enabled_protocol_extensions(cs->_enabled_protocol_extensions) {}
+            : _keyspace(cs->_keyspace)
+            ,  _user(cs->_user)
```

Comment:
An extra space snuck in before _user, increasing latency.

Reply:
done, the latency regression is fixed